### DISCUSSION TITLE
创建合并转发的一个小问题

### DISCUSSION THREAD

**Discussion by Admsec** [takayama-lily/oicq#390] [Q&A]
makeForwardMsg(fake)里面要带的 fake 是什么类型的元素？可以详细一点点说下怎么用吗？我看着文档和源码测试了半天没弄明白怎么创建合并转发🤔
这个是我测试的时候写的，一直都是报错
![image](https://user-images.githubusercontent.com/86643414/179893978-fc94b761-73e6-4ae7-a7cc-fbb2ddc797b6.png)

**Comment by Admsec** [comment]
还有一张图是控制台的报错内容
![image](https://user-images.githubusercontent.com/86643414/179894366-be40119a-fd0c-4020-b1eb-11890bd7de1f.png)

**Comment by dmlgzs** [comment]
建议参考隔壁 Icalingua 里的合并转发

**Comment by dmlgzs** [comment]
https://oicqjs.github.io/oicq/interfaces/Forwardable.html

**Reply by dmlgzs** [reply]
这个是文档的

**Comment by Admsec** [comment]
似乎还是没搞明白这个，算了我太菜了，先放一边
